docs/releases/v1.1.6.md

```
@@ -0,0 +1,16 @@
+## What’s new
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)